rs/sns/testing/BUILD.bazel

```diff
@@ -97,9 +99,20 @@ DEV_ENV = {
 }
 
 rust_binary(
-    name = "cli",
+    name = "sns-testing",
     testonly = True,
-    srcs = ["src/main.rs"],
+    srcs = ["src/bin/main.rs"],
+    data = DEV_DATA,
+    env = DEV_ENV,
+    deps = DEPENDENCIES + [
+        ":sns_testing",
+    ],
+)
+
+rust_binary(
+    name = "sns-testing-init",
```

Comment:
Nit, what we we call this just `sns-init`? 

then the user would need to run `sns-init`, then `sns-testing`, which seems slightly cleared that `sns-testing-init`, which could be confused with a (non-existent) `sns-init-testing` command.

Reply:
Then I think it's better to call this `nns-init` since the binary essentially inits the NNS

Reply:
Unfortunately, we also have an nns-init crate, so let's not add any more confusion. :)

Reply:
Fair enough